.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+  - "12"
   - "10"
   - "8"
   - "6"
```

Comment:
Node.js 6 can be dropped.

Reply:
Hi @StarpTech ^^
I didn't want to take responsibility for this since it requires bumping a major version.
Seen the statement in this PR https://github.com/fastify/fastify-redis/pull/30#issuecomment-488366464 and the fact that fastify 2.x supports Node 6 I thought it would be preferable to delegate this decision to someone else. :smile:
(`Allowing edits from maintainers` option is enabled on my side)